index.html

```diff
@@ -5,8 +5,32 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link href="./styles/index.css" rel="stylesheet">
 </head>
 <body>
-    
+    <h1>Weather Report</h1>
+    <h2 id="cityheader">Current Weather for Seattle</h2>
+    <input type="text" placeholder="Enter a city" id="cityname">
+    <br/>
+    <button onclick="getCurrentTemp()">Get Temperature</button>
+    <button onclick="resetCity()">Reset City</button>
```

Comment:
Yes! Another (and my personally preferred) way to make event listeners is to use the `onclick` event in the HTML element. Wonderful! 🐁 🪤